.konflux/operator/redhat.repo

```diff
@@ -0,0 +1,13 @@
+[rhel-9-for-x86_64-baseos-rpms]
+name = Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)
+baseurl = https://cdn.redhat.com/content/dist/rhel9/9.4/x86_64/baseos/os
```

Comment:
I don't think renovate or mintmaker would update the rhel version in case of updates filed to dockerfile base images. you will need to keep an eye on this file and update the version manually unless you go with an input file with the repos stated and a dockerfile param like this:
https://github.com/openshift-kni/cnf-features-deploy/blob/master/cnf-tests/.konflux/rpms.in.yaml#L14